### PR TITLE
Add support for quantilesDoublesSketch

### LINKED ISSUE #458: quantilesDoublesSketch support
According to [Druid docs](https://druid.apache.org/docs/latest/development/extensions-core/approximate-histograms.html), the Approximate Histogram aggregator is deprecated.

`quantilesDoublesSketch` support was introduced in Plywood 0.20.8, so it should be easy to add to Turnilo.

### PR DESCRIPTION
**Pull request by mkuthan**
Add support for quantilesDoublesSketch during introspection and deduction. 
Closes #458

### CODE REVIEW

File: src/common/models/measure/measure.ts [hunk 1]
```diff
@@ -69,19 +69,24 @@ export class Measure extends BaseImmutable<MeasureValue, MeasureJS> {
...
-  /**
-   * Look for all instances of countDistinct($blah) and return the blahs
-   * @param ex
-   * @returns {string[]}
-   */
-  static getCountDistinctReferences(ex: Expression): string[] {
-    let references: string[] = [];
+  static isCountDistinctReferences(ex: Expression): boolean {
+    let isCountDistinct = false;
     ex.forEach((ex: Expression) => {
```
Comment (adrianmroz) [inline src/common/models/measure/measure.ts]: use `ex.some`

File: src/common/models/measure/measure.ts [hunk 2]
```diff
@@ -69,19 +69,24 @@ export class Measure extends BaseImmutable<MeasureValue, MeasureJS> {
...
   }
 
-  /**
-   * Look for all instances of countDistinct($blah) and return the blahs
-   * @param ex
-   * @returns {string[]}
-   */
-  static getCountDistinctReferences(ex: Expression): string[] {
-    let references: string[] = [];
+  static isCountDistinctReferences(ex: Expression): boolean {
```
Comment (adrianmroz) [inline src/common/models/measure/measure.ts]: `hasCountDistinctSubExpression`?

File: src/common/models/measure/measure.ts [hunk 3]
```diff
@@ -190,7 +195,7 @@ export class Measure extends BaseImmutable<MeasureValue, MeasureJS> {
     // Expression.some is bugged
     let isApproximate = false;
     this.expression.forEach((exp: Expression) => {
-      if (exp instanceof CountDistinctExpression || exp instanceof QuantileExpression) {
+      if (Measure.isCountDistinctReferences(exp) || Measure.isQuantileReferences(exp)) {
```
Comment (adrianmroz) [inline src/common/models/measure/measure.ts]: I think we could remove `this.expression.forEach` and use those two methods. In your code you're doing DFS inside DFS!

File: src/common/models/measure/measure.ts (hunk 1 above)
Comment (mkuthan) [reply src/common/models/measure/measure.ts]: are you sure?
https://github.com/allegro/turnilo/blob/7e75477dadd97192d4614eb51a383983ad05b31a/src/common/models/measure/measure.ts#L195

Comment (adrianmroz) [reply src/common/models/measure/measure.ts]: Maybe we should reimplement this as some util? `let` + `forEach` always makes me worried about future.

**Pull request merged**